doc/whats_new/v0.20.rst

```diff
@@ -156,6 +156,10 @@ Model evaluation and meta-estimators
   group-based CV strategies. :issue:`9085` by :user:`Laurent Direr <ldirer>`
   and `Andreas Müller`_.
 
+- A paramenter `predict_params` was added to :class:`pipeline.Pipeline` allowing
```

Comment:
it's not really a parameter. How about "Pipeline's predict method now passes keyword arguments on to the pipeline's last estimator, enabling the use of `return_std` in a Pipeline (with caution)."? Single backticks do nothing by themselves in our documentation, btw. You need double backtics.